### PR TITLE
access_vectors: update to Xen 4.11

### PR DESCRIPTION
**Pull request by jean-edouard**
Signed-off-by: Jed <lejosnej@ainfosec.com>

### CODE REVIEW

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1790

File: policy/modules/xen/stubdom.if [hunk 1]
```diff
@@ -88,8 +88,7 @@ interface(`stubdom_ioemu',`
 
 	allow stubdom_t $1:resource { add remove };
 	allow stubdom_t $1:domain { getdomaininfo set_target shutdown settime };
-	allow stubdom_t $1:hvm { cacheattr getparam dm
-				 setparam hvmctl };
+	allow stubdom_t $1:hvm { getparam dm setparam hvmctl };
```
Comment (jandryuk) [inline policy/modules/xen/stubdom.if:91]: `cacheattr` goes away because it is now covered by `dm`?

Comment (jean-edouard) [reply policy/modules/xen/stubdom.if:91]: Yes

**Review by jandryuk — APPROVED** [review]

**Comment by jean-edouard** [comment]
Built here: http://openxt-builder.ainfosec.com:8010/builders/openxt/builds/1872

**Comment by jandryuk** [comment]
@eric-ch We need this merged to go along with https://github.com/OpenXT/xenclient-oe/pull/1069 , right?

**Comment by eric-ch** [comment]
@jandryuk Yes, my apologies I thought this was merged as well on Friday.

**Comment by jandryuk** [comment]
No worries.  Thanks!

**Pull request merged**